app/controllers/api/v2/cars_controller.rb

```diff
@@ -0,0 +1,47 @@
+class Api::V2::CarsController < ApplicationController
+  before_action :authorized
+  before_action :set_car, only: %i[show update destroy]
+
+  def index
+    @cars = Car.all
+    render json: @cars
+  end
+
+  def show
+    render json: @car
+  end
```

Comment:
- [ ] With the current changes made, a user can view a car alright but only cars that belong to the user. In the front when a user visits the home to rent a car, he should be able to have access to the show view of the car even if the car doesn't belong to him. I suggest that the show method should not have a `before_action:set_car`. This will allow all users to view all cars even if they don't belong to them in order to rent it.